tools/rvn/CommandLineApp.cs

```diff
@@ -15,6 +21,9 @@ internal static class CommandLineApp
 
         private static CommandLineApplication _app;
 
+        private const string LetsEncryptMode = "lets-encrypt";
+        private const string ImportCertificateMode = "import-certificate";
```

Comment:
So we'll have:

`rvn create-setup-package lets-encrypt`
`rvn create-setup-package import-certificate {path-to-cert}` 


Is that correct? 

I have some doubts if `import-certificate` is the correct name for that setup mode. Other options: `with-certificase`, `self-signed-certificate` or just `secured` like the enum value of `SetupMode`?

Thoughts? CC @gregolsky

Reply:
And just to make sure - those options will be printed to the user when --help will be used, right?

Reply:
@arekpalinski From what I see it's gonna be:

```
rvn create-setup-package --mode lets-encrypt --setup-parameters-path setup.json
rvn create-setup-package -m lets-encrypt -s setup.json
rvn create-setup-package import-certificate --cert-path cert.pfx
```

Reply:
Maybe `own-certificate`?

Reply:
I think `own-certificate` suits better than `import-certificate`

Reply:
@arekpalinski  regarding the `enum`.
Creating an `enum` will cause me to add `[Description]` property which will cause trying to parse the string value I'm getting from the user. 
I couldn't make it work. I think the better choice, in this case, is just using two consts.

Am I missing something here ?

Reply:
okay, if that causes too much hassle then let's keep it simple with the usage of const